### PR TITLE
Dev server

### LINKED ISSUE #33: add dev server
I think add a script just like `npm run dev`. It can see the example and it's convenient to debug and coding lozad.js

### PR DESCRIPTION
**Pull request by stephenkingsley**
fix #33 

you can run `npm run dev`，and see the example

### CODE REVIEW

**Comment by stephenkingsley** [comment]
@thiamsantos @stephenkingsley @lptn @bfred-it please review

File: example/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
```
Comment (ApoorvSaxena) [inline example/index.html]: @stephenkingsley no point of having a seperate example, when we already have demo

File: example/server.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const path = require('path')
```
Comment (ApoorvSaxena) [inline example/server.js]: what's the need of creating a separate `node server` and `webpack`, when we are already making use of `rollup`? Read about [Rollup watch](https://rollupjs.org/#rollup-watch)

File: example/index.html (hunk 1 above)
Comment (ApoorvSaxena) [reply example/index.html]: we can temporarily remove the CDN link in demo for testing purposes

File: example/server.js (hunk 2 above)
Comment (stephenkingsley) [reply example/server.js]: I think our demo's javascript script depended online version. And I think this dev server is depend our code in src. It is convenient to debug and develop. What do you think? I think demo should update so I created example.

Comment (ApoorvSaxena) [reply example/server.js]: we can simply reuse demo for testing purposes as well

Comment (stephenkingsley) [reply example/server.js]: you have to change the js cdn. In my opinions, it is not necessary. Just run `npm run dev` and then see the demo and testing is very cool!

Comment (ApoorvSaxena) [reply example/server.js]: @stephenkingsley will be happy to merge if you reuse existing demo code and integrate in `rollup` as advised above

Comment (stephenkingsley) [reply example/server.js]: Ok, I think @sangle7 can do this! Reuse existing demo and delete example folder.

Comment (ApoorvSaxena) [reply example/server.js]: @stephenkingsley closing this pull request for now

Comment (stephenkingsley) [reply example/server.js]: @ApoorvSaxena Ok, I will open another pr

**Pull request closed without merging**